solr/solrj/src/java/org/apache/solr/client/solrj/impl/CloudLegacySolrClient.java

```diff
@@ -270,6 +271,17 @@ public Builder withLBHttpSolrClient(LBHttpSolrClient loadBalancedSolrClient) {
       return this;
     }
 
+    /**
+     * Sets the cache ttl for DocCollection Objects cached.
+     *
+     * @param seconds ttl value in seconds
+     */
+    public Builder setCollectionCacheTtl(int seconds) {
```

Comment:
`with`

Reply:
sent a inquery to the mailing list and a mention on the jira...   Let's see if that shakes any opinons loose....  Please do speak up!

Reply:
I'm going to go with `with` for now, as I'm anxious to get this committed.   There is a seperate JIRA out to ask for more global feedback....   and this could always be changed at that point ;-)